pkg/transformer/kubernetes/kubernetes_test.go

```diff
@@ -511,3 +512,14 @@ func TestInitPodSpec(t *testing.T) {
 		t.Fatalf("Pod object not found")
 	}
 }
+
+func TestKubernetes_ConfigTmpfs(t *testing.T) {
```

Comment:
One last change here ;-)
Can you please rename this? Its in Kubernetes package so it doesn't have to to have Kubernetes in name and `_` is not GOlangy :-)

just `TestConfigTmpfs` says everything about what it is doing